modules/library/jdbc/src/main/java/org/geotools/jdbc/SQLDialect.java

```diff
@@ -1247,6 +1247,10 @@ public void createIndex(Connection cx, SimpleFeatureType schema, String database
             sql.append("UNIQUE ");
         }
         sql.append("INDEX ");
+        if (databaseSchema != null) {
+            encodeSchemaName(databaseSchema, sql);
+            sql.append(".");
+        }        
         sql.append(escape).append(index.getIndexName()).append(escape);
```

Comment:
Please update the header to  (C) 2002-2015 when making a change.

This change does not have a test case so I cannot determine if it is "ok". Is it already covered by a DB2TestCase - if so which one?